fivegmag_5GMSdAwareApplication/app/src/main/java/com/fivegmag/a5gmsdawareapplication/MainActivity.kt

```diff
@@ -322,7 +324,11 @@ class MainActivity : AppCompatActivity(), AdapterView.OnItemSelectedListener {
     }
 
     private fun onM8DataChanged() {
-        mediaSessionHandlerAdapter.setM5Endpoint(m8Data.m5BaseUrl)
+        val configPropertiesDefualtTimer : Properties = loadConfiguration("config.propertiesTimer.xml")
+
+        val defaultTimerVal: Long = configPropertiesDefualtTimer.getProperty("defaultServiceAccessInformationTimerVal").toLong()
+
+        mediaSessionHandlerAdapter.setM5Endpoint(m8Data.m5BaseUrl, defaultTimerVal)
```

Comment:
Just an observation based on my quick review: It seems odd to me that the 5GMS-Aware Application would have a concept of a default Service Access Information timer, or that the Media Session Handler would expose a parameter to the application allowing it to set this value.

The time-to-live of the Service Access Information is normally indicated in the M5 response to the Service Access Information request. If you think it's necessary to have a default time-to-live value (in case the M5 response lacks the necessary cache control response headers to signal it) I think it would be more appropriate to make this a configuration parameter of the Media Session Handler than of the 5GMS-Aware Application.

Passing a default value in from the application feels like it goes beyond the M6 API abstraction. I don't think the application needs to worry itself with this level of detail.

Reply:
I agree on this, @shilinding please add this parameter to the Media Session Handler. Maybe my comment here https://github.com/5G-MAG/rt-5gms-media-session-handler/pull/55#discussion_r1474669655 was a bit misleading. I only referenced the 5GMS Aware Application configuration file as an example.

Reply:
Thanks, @shilinding. Looks good.